src/main/java/thePackmaster/actions/graveyardpack/ReturnExhaustedToDeckAction.java

```diff
@@ -0,0 +1,141 @@
+package thePackmaster.actions.graveyardpack;
+
+import com.megacrit.cardcrawl.actions.AbstractGameAction;
+import com.megacrit.cardcrawl.actions.common.MakeTempCardInDrawPileAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.characters.AbstractPlayer;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.localization.UIStrings;
+import com.megacrit.cardcrawl.relics.AbstractRelic;
+
+import java.util.ArrayList;
+import java.util.Iterator;
+
+import static thePackmaster.SpireAnniversary5Mod.makeID;
+
+public class ReturnExhaustedToDeckAction
```

Comment:
I haven't looked at this, I'll just ask if this can use either `ExhaustToHandAction` (takes a card) or `ExhumeAction` (like exhume) instead of making this

Reply:
I think this needs to be custom, both because ExhumeAction really wants to put the card in your hand, and to exclude non-class cards.

Reply:
Nevermind, I replaced it with MoveCardsAction plus a shuffle.